examples/SingleNeuron_HH1952/main.cpp

```diff
@@ -116,6 +116,9 @@ class HH_Neuron : public Neuron {
 };
 
 int main(int argc, char **argv) {
+  double default_simulation_time = 50.0;
+  insilico::engine::simulation_time = default_simulation_time;
```

Comment:
You could directly use `engine::simulation_time` instead of referring is `insilico::`, as you are telling the compiler about which namespace you are in, above, with `using namespace insilico`.